#include "gsteam_normal.h"
#include "src/network/rpc_closure.h"
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE

///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE

///<<<rpc begin
void ServerTeamServiceImpl::EnterScene(::google::protobuf::RpcController* controller,
    const normal::TeamTestRequest* request,
    normal::TeamTestResponse* response,
    ::google::protobuf::Closure* done)
{
    AutoRecycleClosure d(done);
///<<< BEGIN WRITING YOUR CODE EnterScene
///<<< END WRITING YOUR CODE EnterScene
}

///<<<rpc end
